[2025-08-13T13:03:55.553+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_pipeline.transform_data manual__2025-08-13T13:03:50.799378+00:00 [queued]>
[2025-08-13T13:03:55.567+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_pipeline.transform_data manual__2025-08-13T13:03:50.799378+00:00 [queued]>
[2025-08-13T13:03:55.568+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-08-13T13:03:55.587+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform_data> on 2025-08-13 13:03:50.799378+00:00
[2025-08-13T13:03:55.593+0000] {standard_task_runner.py:57} INFO - Started process 180 to run task
[2025-08-13T13:03:55.599+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_pipeline', 'transform_data', 'manual__2025-08-13T13:03:50.799378+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl.py', '--cfg-path', '/tmp/tmp2f8_vu6o']
[2025-08-13T13:03:55.604+0000] {standard_task_runner.py:85} INFO - Job 49: Subtask transform_data
[2025-08-13T13:03:55.627+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:193 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-13T13:03:55.681+0000] {task_command.py:416} INFO - Running <TaskInstance: weather_etl_pipeline.transform_data manual__2025-08-13T13:03:50.799378+00:00 [running]> on host b380b9cae19f
[2025-08-13T13:03:55.781+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl_pipeline' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-13T13:03:50.799378+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-13T13:03:50.799378+00:00'
[2025-08-13T13:03:55.806+0000] {logging_mixin.py:151} INFO - Transformed CSV saved to /opt/***/data/weather_data_transformed.csv
[2025-08-13T13:03:55.807+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-08-13T13:03:55.823+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=weather_etl_pipeline, task_id=transform_data, execution_date=20250813T130350, start_date=20250813T130355, end_date=20250813T130355
[2025-08-13T13:03:55.866+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-08-13T13:03:55.899+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
